docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -118,25 +146,137 @@ To retrieve the metadata and playback information on the client side:
+
+   ```
+   mc_client_destroy(g_client_h);
+   ```
+
+<a name="get_playlist"></a>
+## Updating and Retrieving Playlist
+
+To update the playlist and metadata information on the server side:
+
+1. Create the media controller server handle using the `mc_server_create()`:
+
+   ```
+   ret = mc_server_create(&g_server_h);
+   ```
+2. Create the playlist handle using the `mc_server_create_playlist()`:
+
+   ```
+   mc_playlist_h playlist_h = NULL;
+
+   ret = mc_server_create_playlist(g_server_h, "playlist", &playlist_h);
+   ```
+3. Set the metadata information in the playlist handle to be updated using the corresponding `mc_server_add_item_to_playlist()` function, and then update the playlist using the corresponding `mc_server_update_playlist_done()` function:
+
+   ```
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_TITLE, "title_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_ARTIST, "artist_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_ALBUM, "album_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_AUTHOR, "author_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_GENRE, "genre_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_DURATION, "duration_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_DATE, "date_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_COPYRIGHT, "copyright_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_DESCRIPTION, "desc_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_TRACK_NUM, "tracknum_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_PICTURE, "picture_1");
+
+   ret = mc_server_update_playlist_done(g_server_h, playlist_h);
+   ```
+4. When no longer needed, destroy the playlist handle using the `mc_playlist_destroy()`:
+
+   ```
+   mc_playlist_destroy(playlist_h);
+   ```
+5. Destroy the media controller server handle using the `mc_server_destroy()`, when media controller server handle is no longer needed:
+
+   ```
+   mc_server_destroy(g_server_h);
+   ```
+
+To retrieve the playlist and metadata information on the client side:
+
+1. Create the media controller client handle using the `mc_client_create()`:
+
+   ```
+   ret = mc_client_create(&g_client_h);
+   ```
+2. Define the callback that is invoked when the client receives the playlist:
+
+   ```
+   void
+   playlist_updated_cb(const char *server_name, mc_playlist_update_mode_e mode, const char *playlist_name, mc_playlist_h playlist, void *user_data)
+   {
+     dlog_print(DLOG_DEBUG, LOG_TAG, "Server Name: %s, Playlist Update Mode: %d, Playlist Name: %s\n", server_name, mode, playlist_name);
```

Comment:
For proper indentation, please use 4 spaces in the next line from the place of "{".

Reply:
OK, it will be changed.